x/ccv/provider/keeper/keeper.go

```diff
@@ -1513,3 +1514,38 @@ func (k Keeper) IsDenylistEmpty(ctx sdk.Context, chainID string) bool {
 
 	return !iterator.Valid()
 }
+
+// GetLastBondedValidators iterates the last validator powers in the staking module
+// and returns the first MaxValidators many validators with the largest powers.
+func (k Keeper) GetLastBondedValidators(ctx sdk.Context) []stakingtypes.Validator {
+	var lastPowers []stakingtypes.LastValidatorPower
```

Comment:
nit: `lastPowers` can be initiated with a length set to `maxVals`.

Reply:
Sure, can change